pkg/filters/redirectorv2/redirectorv2.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/megaease/easegress/v2/pkg/context"
 	"github.com/megaease/easegress/v2/pkg/filters"
+	httprouters "github.com/megaease/easegress/v2/pkg/object/httpserver/routers"
```

Comment:
Looks good to me. Just a little advice. How about add a type alias in `github.com/megaease/easegress/v2/pkg/protocols/httpprot` say:

```
var Route = github.com/megaease/easegress/v2/pkg/object/httpserver/routers.Route
```
then 
```
prefix := route.(httpprot.Route).GetPathPrefix()
```
The code produce exactly same result. But for future developers, it may a little bit easier for them to find right interface?

Reply:
That will cause an import cycle.